libs/core-ui/src/lib/Interfaces/IDataset.ts

```diff
@@ -31,6 +31,8 @@ export interface IDataset {
   feature_metadata?: IFeatureMetaData;
   images?: string[];
   index?: string[];
+  object_detection_true_y?: number[][][];
```

Comment:
@imatiach-msft where are these python args passed to the typescript interfaces? Wish to emulate something similar:

![image](https://user-images.githubusercontent.com/37190647/225452660-199ea112-b4ea-4836-80cb-8da8f5649c9c.png)

![image](https://user-images.githubusercontent.com/37190647/225452755-1b2af296-c9b3-4f54-b2ac-e7fc615b3b34.png)

Also the changes may involve defining a new interface, parsing python list to that interface, and including that interface in unit tests so I propose doing that in the next PR to avoid overloading this one.

Reply:
Thanks for investigating this! My concern here is that when implementing the explainability logic, I was planning on relying on the indexes of the bounding boxes. Would that still be possible with a dictionary representation? What would be the keys of the dictionary?